runtime/src/main/java/org/corfudb/runtime/object/transactions/OptimisticTransactionalContext.java

```diff
@@ -271,7 +271,7 @@ public long getConflictSetAndCommit(ConflictSetInfo conflictSet) {
         // Write to the transaction stream if transaction logging is enabled
         Set<UUID> affectedStreamsIds = new HashSet<>(getWriteSetInfo().getWriteSet().getEntryMap().keySet());
 
-        if (this.builder.runtime.getObjectsView().isTransactionLogging()) {
+        if (this.transaction.runtime.getObjectsView().isTransactionLogging()) {
```

Comment:
would be nice to provide `this.transaction.isTransactionLogging()`

Reply:
transaction logging is not per transaction, its runtime wide.

Reply:
Its much clean now. Thanks!